crates/bevy_ui/src/geometry.rs

```diff
@@ -356,39 +355,55 @@ impl Size {
     };
 }
 
-impl Add<Vec2> for Size {
+impl Add<(Val, Val)> for Size {
```

Comment:
Taking a look, I don't like these impls (or the previous ones for Vec2).

Addition and subtraction are common, hard-to-grep operations: they should not implicitly panic.

Instead, I would just remove these, and add a `From<(Val, Val)> for Size` impl if it doesn't exist already.